core/chainio/avs_writer.go

```diff
@@ -70,7 +72,10 @@ func NewAvsWriterFromConfig(baseConfig *config.BaseConfig, ecdsaConfig *config.E
 	}, nil
 }
 
-func (w *AvsWriter) SendAggregatedResponse(batchIdentifierHash [32]byte, batchMerkleRoot [32]byte, senderAddress [20]byte, nonSignerStakesAndSignature servicemanager.IBLSSignatureCheckerNonSignerStakesAndSignature) (*common.Hash, error) {
+// Sends AggregatedResponse and waits for the receipt for three blocks, if not received
+// it will try again bumping the last tx gas price based on `CalculateGasPriceBump`
+// This process happens indefinitely until the transaction is included.
```

Comment:
Maybe for a separate PR, but we need to add some numbers to graph the retry logic. See https://github.com/yetanotherco/aligned_layer/blob/b03a625a2fc755459797d013297de83793521a48/batcher/aligned-batcher/src/lib.rs#L1422